samples/ContainerRun/Program.cs

```diff
@@ -47,6 +50,12 @@ static void Main(string[] args)
                             process.WaitForExit(5000);
                             Console.Out.WriteLine("process exited with {0}", process.ExitCode);
                         }
+                        }
+                        finally
+                        {
+                            Console.Out.WriteLine("shutting down container");
+                            container.Shutdown(Timeout.Infinite);
```

Comment:
We chatted, and are going to use a default parameter value instead.  Language features for the win!